templates/visuals/slicer/src/settings.ts

```diff
@@ -26,7 +26,33 @@
 
 "use strict";
 
-import { dataViewObjectsParser } from "powerbi-visuals-utils-dataviewutils";
-import DataViewObjectsParser = dataViewObjectsParser.DataViewObjectsParser;
+import { formattingSettings } from "powerbi-visuals-utils-formattingmodel";
 
-<%= visualSettingsClasses %>
+import FormattingSettingsCard = formattingSettings.Card;
+import FormattingSettingsSlice = formattingSettings.Slice;
+import FormattingSettingsModel = formattingSettings.Model;
+
+
+/**
+ * General Formatting Card
+ */
+class generalCardSettings extends FormattingSettingsCard {
+    filter: FormattingSettingsSlice = undefined;
```

Comment:
I added here undefined cards because it was the same (slices values are undefined) when i ran` pbiviz new -t slicer`